multimodal/src/autogluon/multimodal/models/timm_image.py

```diff
@@ -87,21 +91,35 @@ def image_feature_dim(self):
 
     def forward(
         self,
-        batch: dict,
+        images: torch.FloatTensor,
+        image_valid_num: torch.Tensor,
+        image_column_names: Optional[List[str]] = None,
+        image_column_values: Optional[List[torch.Tensor]] = None,
```

Comment:
How about changing the name `image_column_values `  to `image_column_indexes`? It stores the start and end indexes of each column's images in `images`.

Reply:
Renamed to `image_column_indices`.